app/routes/onboardingUser/connect.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC } from 'react';
+import { ActionArgs, json, redirect } from '@remix-run/node';
+import { register } from '~/api/onboarding/register.server';
+import { FormTitlesAndSubtitles } from '~/constants/userOnboarding';
+import ContentPill from '~/components/common/contentPill';
+import { GoogleCalendar, MicrosoftCalendar } from '~/components/icons';
+
+interface UserDetailsInterface {
+  apiHost: string;
+  authToken: string;
+}
+
+export async function loader() {
+  return json({ apiHost: process.env.API_HOST, authToken: process.env.AUTH_TOKEN });
```

Comment:
we should put `process.env.API_HOST` and `process.env.AUTH_TOKEN` in constants and use a variable everywhere instead.